.env-dist

```diff
@@ -44,3 +44,7 @@ HIBP_RELOAD_BREACHES_TIMER=600
 # HIBP API for range search and subscription
 HIBP_KANON_API_ROOT="https://api.haveibeenpwned.com"
 HIBP_KANON_API_TOKEN=""
+# How many seconds to wait before retrying an HIBP request
```

Comment:
Sorry, small nit/question...

Did you mean "seconds" or "milliseconds"? I believe the code below implies milliseconds, whereas if it's seconds, we're talking about a 33.33 minute backoff.